docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single-node.md

```diff
@@ -0,0 +1,765 @@
+---
+content_title: "Tic-tac-toe Game Contract Using Single Node"
+link_text: "Tic-tac-toe Game Contract Using Single Node"
+---
+
+# Your Guide to Tic-tac-toe on EOSIO 
+This tic-tac-toe tutorial guides you step by step to build a tic-tac-toe game which runs on an EOSIO blockchain. You will create a smart contract containing the game logic, then compile and deploy this smart contract to an EOSIO blockchain. In this tutorial we use a local [single node testnet](https://developers.eos.io/manuals/eos/latest/nodeos/usage/development-environment/local-single-node-testnet) and show you how to play the game by calling the smart contract. For another example in using the single node testnet see the [Getting Started](https://developers.eos.io/welcome/latest/getting-started/index) section.
+
+[[info]]
+| EOSIO is a blockchain software platform developed by block.one. Smart contracts which run on the EOSIO platform should run on any blockchain which using the EOSIO platform. EOS is a digital token, though it also refers to the first public blockchain run on the EOSIO platform, EOS or the public mainnet. In this tutorial we will use the EOSIO platform to run a blockchain on your local machine. This runs only one producing node and is commonly called a single node testnet.
+
+
+We explain the purpose of each step and why it is important.
+
+## Prerequisites for the Tutorial
```

Comment:
We can just say **Prerequisites**.

Suggestions for this section:

This tutorial requires the following tools and knowledge:

**Software**

* The EOSIO platform software. For installation instructions, see [Install Prebuilt Binaries](https://developers.eos.io/manuals/eos/latest/install/install-prebuilt-binaries).
* EOSIO CDT for smart contract compilation. For installation instructions, see the [CDT Installation](https://developers.eos.io/manuals/eosio.cdt/latest/installation) section.

**Knowledge and Familiarity**

* Familiar with EOSIO protocols such as Actions, Transactions, Accounts, and Permissions.
* Familiar with basic EOSIO Smart Contract development. To understand the sequence and gain functional knowledge of EOSIO smart contract development, see the [Getting Started](https://developers.eos.io/welcome/latest/getting-started/index) Guide and build a basic [Hello World Smart Contract](https://developers.eos.io/welcome/latest/getting-started/smart-contract-development/hello-world.)
* Intermediate to proficient level of C++ programming and object oriented programming

**Tools**

* A code editor or an IDE of choice

For additional environment requirements, see the Prerequisites information in the Getting Started section.